### PR TITLE
Fix Address#country_subdevision_code

### LINKED ISSUE #23: Fix Address#country_subdevision_code spelling
As noted by @Drinion [here](https://github.com/fadendaten/solidus_six_saferpay/pull/16#discussion_r699929350), the `country_subdivision_code` attribute of the `Address` model contains a spelling mistake that should be fixed. Since the spelling is consistent throughout the project this is also passed to the SIX api and therefore it's currently impossible to submit an actual value for the country subdivision code via this gem.

### PR DESCRIPTION
**Pull request by mohitsaxenaknoldus**
Fixes #23

### CODE REVIEW

File: spec/fixtures/vcr_cassetts/secure_pay_gate_create_offer.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ http_interactions:
       string: '{"Payment":{"Amount":{"Value":100,"CurrencyCode":"CHF"},"OrderId":"R123456789","Description":"Order
         R123456789"},"ExpirationDate":"2019-11-16","Payer":{"IpAddress":"127.0.0.1","LanguageCode":"de-ch","BillingAddress":{"FirstName":"John","LastName":"Doe","DateOfBirth":"1990-05-31","Company":"ACME
         Corp.","Gender":"COMPANY","LegalForm":"AG","Street":"Bakerstreet 32","Street2":"Stewart
-        House","Zip":"8000","City":"Zurich","CountrySubdevisionCode":"ZH","CountryCode":"CH","Phone":"+41
+        House","Zip":"8000","City":"Zurich","CountrysubdivisionCode":"ZH","CountryCode":"CH","Phone":"+41
```
Comment (Drinion) [inline spec/fixtures/vcr_cassetts/secure_pay_gate_create_offer.yml]: @mohitsaxenaknoldus Please change `CountrysubdivisionCode` to `CountrySubdivisionCode`. Thanks!

**Review by Drinion — COMMENTED** [review]
Thanks for your help!

**Comment by mohitsaxenaknoldus** [comment]
Done @Drinion

**Comment by Drinion** [comment]
@mohitsaxenaknoldus Thanks. @jugglinghobo IMO good to merge.

**Comment by knorrli** [comment]
@Drinion go ahead :+1:, I'll test this with https://github.com/fadendaten/solidus_six_saferpay soon

@mohitsaxenaknoldus thank you very much for your contribution!

**Pull request merged**